src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -979,6 +979,9 @@ because xbuild doesn't support framework reference assemblies.
 	<_AndroidResourceDesignerFile Condition=" '$(AndroidUseIntermediateDesignerFile)' != 'True' ">$(AndroidResgenFile)</_AndroidResourceDesignerFile>
 	<_AndroidStaticResourcesFlag>$(IntermediateOutputPath)static.flag</_AndroidStaticResourcesFlag>
 	<_AndroidResourcesCacheFile>$(IntermediateOutputPath)mergeresources.cache</_AndroidResourcesCacheFile>
+	<_AndroidUseManagedDesignTimeResourceGenerator Condition=" '$(_AndroidUseManagedDesignTimeResourceGenerator)' == '' " >True</_AndroidUseManagedDesignTimeResourceGenerator>
```

Comment:
Should this be "private"? Don't we want this to be overridable in `App.csproj` in case it breaks things, and otherwise be documented? It seems like we really want `$(AndroidUseManagedDesignTimeResourceGenerator)` ("public"), and a corresponding entry within `Documentation/build_process.md`.